tasks/main.yml

```diff
@@ -1,17 +1,12 @@
 ---
-- name: Perform parameter compatibility check
+- name: Check if Subscription Manager is about to be used on non RHEL system
   fail:
-    msg: "Subcription manager could be used only on Red Hat Enterprise Linux"
-  when: ovirt_repositories_use_subscription_manager and not (ansible_distribution == 'RedHat')
+    msg: "Subscription manager could be used only on Red Hat Enterprise Linux"
+  when: ovirt_repositories_use_subscription_manager and ansible_distribution != 'RedHat'
```

Comment:
Wouldn't it be safer to use `when: ovirt_repositories_use_subscription_manager|bool ...`?

Reply:
Done